docs/guides/windows_service.md

```diff
@@ -3,71 +3,75 @@ layout: default
+*This is the FDM Monster web app after installation (visit [http://localhost:4000](http://localhost:4000))*
 
-## Preparation of the Windows service installation
+## Preparation for the Windows service installation
+
+The following steps will install:
 
-The following steps will install the following:
 - nodejs 16+
 - git
 - yarn (npm package)
 - fdm-monster (github cloned source code)
 - node-windows (npm package)
 
-At step 4 you **can** choose to skip the remaining steps by running the `installations/fdm-monster-node-windows` installation scripts.
-Step 5 to 8 are just there to let you know how the scripts work. Step 9 concludes the installation.
+At step 4, you **can** choose to skip the remaining steps by running the `installations/fdm-monster-node-windows` installation scripts. Step 5 to 8 are there to explain how the scripts work. Step 9 concludes the installation.
 
 ### Caveats
 
-- Internet is required
-- Windows only (please use docker on Linux)
-- Pay attention to versions (f.e. Node 16 or 18, and not Node 15/17)
-- Understand powershell execution policy: `Set-ExecutionPolicy -ExecutionPolicy Unrestricted` will allow anything to run.
-
+- Internet access is required
+- Windows only (if you use Linux, please use Docker)
+- Pay attention to versions (e.g., Node 16 or 18, not Node 15/17)
+- Understand PowerShell execution policy: `Set-ExecutionPolicy -ExecutionPolicy Unrestricted` will allow anything to run.
+---
 ### Step 1) Installing NodeJS 16+
-Install NodeJS LTS (long-term support) from [https://nodejs.org/en/download](https://nodejs.org/en/download). 
-At the moment of writing this is Node 18. The FDM Monster server requires you to use at least NodeJS LTS 16.
 
-To check Node is installed properly, you can execute this in command prompt, or in powershell:
+Install NodeJS LTS (long-term support) from [https://nodejs.org/en/download](https://nodejs.org/en/download). At the time of writing, this is Node 18. The FDM Monster server requires NodeJS LTS 16 or higher.
+
+To check whether Node is installed properly, execute this in Command Prompt or PowerShell:
+
 ```
 node -v
 ```
 
 The output should be:
-> PS C:\Users\SomeUser> node -v
-<br/> v18.14.2
-
+```
+PS C:\Users\SomeUser> node -v
+v18.14.2
+```
+---
 ### Step 2) Installing MongoDB 5+
-Install MongoDB Community Edition. This URL should allow you to download the MongoDB installation setup: 
-[MongoDB Download](https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-6.0.5-signed.msi). 
-If not visit [MongoDB MSI download page](https://www.mongodb.com/try/download/community).
 
+Install MongoDB Community Edition from [https://www.mongodb.com/try/download/community](https://www.mongodb.com/try/download/community). You can use [this URL](https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-6.0.5-signed.msi) to download the MongoDB installation setup if the previous link doesn't work.
+---
```

Comment:
Line 57 and 58 need to be separated. For some reason this will be rendered as big text if there is no newline between `---` and the line before.
![image](https://github.com/fdm-monster/fdm-monster/assets/6005355/db064ab8-d877-491f-a56f-67969906ade5)